doc/classes/ParticlesMaterial.xml

```diff
@@ -330,5 +333,17 @@
 		<constant name="EMISSION_SHAPE_MAX" value="6" enum="EmissionShape">
 			Represents the size of the [enum EmissionShape] enum.
 		</constant>
+		<constant name="EMISSION_POINTS_ORDER_RANDOM" value="0" enum="PointsOrder">
+			Particles from point textures will be emitted with a random order.
+		</constant>
+		<constant name="EMISSION_POINTS_ORDER_NORMAL" value="1" enum="PointsOrder">
+			Particles from points texture will emit in the order in which they are stored in the  [member emission_point_texture].
+		</constant>
+		<constant name="EMISSION_POINTS_ORDER_REVERSE" value="2" enum="PointsOrder">
```

Comment:
Likewise, this should probably be "REVERSE_SEQUENTIAL" (for consistency with other particle enum constants such as REVERSE_LIFETIME).